julia.yaml

```
@@ -0,0 +1,32 @@
+package:
+  name: julia
+  version: 1.9.4
+  epoch: 0
+  description: The Julia Programming Language
+  copyright:
+    - license: MIT
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - perl
+      - python3
```

```suggestion
      - python3
      - llvm16-dev
```

You need `LLVM` to build julia.

And for lining you can use https://github.com/chainguard-dev/yam
It should look like
```YAML
package:
  name: julia
  version: 1.9.4
  epoch: 0
  description: The Julia Programming Language
  copyright:
    - license: MIT

environment:
  contents:
    packages:
      - build-base
      - busybox
      - ca-certificates-bundle
      - llvm16-dev
      - perl
      - python3

pipeline:
  - uses: git-checkout
    with:
      repository: https://github.com/JuliaLang/julia.git
      tag: v${{package.version}}
      depth: 1
      expected-commit: 8e5136fa2979885081cd502d2210633dff1d2a1a

  - uses: autoconf/make

  - runs: |
      make install DESTDIR=${{targets.destdir}} prefix=""

  - uses: strip

update:
  enabled: true
  github:
    identifier: JuliaLang/julia
    strip-prefix: v
```

---

thanks a lot for your support. Let me update the PR.